### PR TITLE
balloon_stress:default maximum balloon value correct and test logic optmize

### PR DESCRIPTION
**Pull request by aliang123**
qemu.tests.balloon_stress:
1.set default maximum balloon value--vm maximum mem
2.set memory balloon randomly, not constant value.

Signed-off-by: Aihua Liang<aliang@redhat.com>

### CODE REVIEW

**Comment by aliang123** [comment]
ID:1390052,1388321

File: qemu/tests/balloon_stress.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def run(test, params, env):
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
 
-    default_memory = int(params.get("default_memory", 8192))
+    default_memory = int(params.get("default_memory", params.get('mem')))
```
Comment (CongLi) [inline qemu/tests/balloon_stress.py]: params['mem']

File: qemu/tests/balloon_stress.py [hunk 2]
```diff
@@ -51,27 +51,14 @@ def run(test, params, env):
     if not running:
         raise error.TestError("Video do not playing")
 
-    env["balloon_test"] = 0
```
Comment (CongLi) [inline qemu/tests/balloon_stress.py]: Please keep it which will be used in balloon_in_use cases.

File: qemu/tests/balloon_stress.py (hunk 1 above)
Comment (aliang123) [inline qemu/tests/balloon_stress.py]: updated, please help to check.

File: qemu/tests/balloon_stress.py (hunk 2 above)
Comment (aliang123) [inline qemu/tests/balloon_stress.py]: Updated, please help to check.

**Comment by CongLi** [comment]
LGTM,
acked-by: Cong Li <coli@redhat.com>

**Comment by xutian** [comment]
looks OK to me.

**Comment by CongLi** [comment]
Thanks @aliang123 @xutian .

**Pull request merged**